core/shared/src/main/scala/monocle/Fold.scala

```diff
@@ -68,22 +67,22 @@ abstract class Fold[S, A] extends Serializable { self =>
     !isEmpty(s)
 
   /** join two [[Fold]] with the same target */
-  @inline final def choice[S1](other: Fold[S1, A]): Fold[S \/ S1, A] =
-    new Fold[S \/ S1, A]{
-      def foldMap[M: Monoid](f: A => M)(s: S \/ S1): M =
+  @inline final def choice[S1](other: Fold[S1, A]): Fold[Either[S, S1], A] =
+    new Fold[Either[S, S1], A]{
```

Comment:
The direct translation of `S \/ S1` is `S Either S1` but I think it would be easier to stick to `Either[S, S1]`